packages/mdc-radio/_mixins.scss

```diff
@@ -393,3 +411,108 @@ $mdc-radio-ripple-target: ".mdc-radio__ripple";
     margin: 0;
   }
 }
+
+///
+/// Helps select the radio background only when its native control is in the
+/// enabled state.
+/// @access private
+///
+@mixin mdc-radio-if-enabled_ {
+  .mdc-radio__native-control:enabled + {
+    @content;
+  }
+}
+
+///
+/// Helps select the radio background only when its native control is in the
+/// enabled & unchecked state.
+/// @access private
+///
+@mixin mdc-radio-if-enabled-unchecked_ {
+  .mdc-radio__native-control:enabled:not(:checked) + {
+    @content;
+  }
+}
+
+///
+/// Helps select the radio background only when its native control is in the
+/// enabled & checked state.
+/// @access private
+///
+@mixin mdc-radio-if-enabled-checked_ {
+  .mdc-radio__native-control:enabled:checked + {
+    @content;
+  }
+}
+
+///
+/// Helps select the radio background only when its native control is in the
+/// disabled state.
+/// @access private
+///
+@mixin mdc-radio-if-disabled_ {
+  [aria-disabled="true"] .mdc-radio__native-control,
```

Comment:
is aria selector necessary?

Reply:
I assumed it was there for a11y reasons. Any reason it may potentially be obsolete?